docs/dynamic-plugins/versions.md

```diff
@@ -1,132 +1,324 @@
-# Version Compatibility Matrix
 
-## RHDH 1.4 (pre-release, versions can change for final release)
+## RHDH next (pre-release, versions can change for final release)
```

Comment:
I trust in the `release-1.y` branches we'll see a real version here, not `next` ? Also given we have https://github.com/redhat-developer/rhdh/blob/main/package.json#L3 could (should?) this instead be derived from that file?

Reply:
> I trust in the release-1.y branches we'll see a real version here, not next ?

I intentionally did not want to do that to make it clear that this is not released yet. That is why I choose to show next instead of version.

> Also given we have https://github.com/redhat-developer/rhdh/blob/main/package.json#L3 could we (should we?) use the versions derived from that file?

It has a full version (including z stream). I did not want to go to that level. But I can start parsing this instead of branch name if that is for some reason better.